addons/core/addon/helpers/doc-url.js

```diff
@@ -23,6 +23,6 @@ export default class extends Helper {
       );
     }
     const path = configuredPath || '';
-    return `${baseURL}${path}`;
+    return /^(http|https):\/\/(\S)+$/.test(path) ? path : `${baseURL}${path}`;
```

Comment:
I just wanted to check, is this to check if this is a URL that starts with an http or https scheme to differentiate between a relative url vs an absolute url?

If that's the case, it seems like we just need to check if the string starts with http like `path.startsWith('http')` (do we even have just `http` links?) or keep the regex but just remove the `(\S)` (I don't think you care about the rest of the url nor does it need to be captured)

Reply:
we only have one https link. I like ur suggestion of using startsWith instead. Ill use startsWith('https') instead since there is no http link. Initially i added http to the regex just in case we do ever add one but now thinking about it that seems unlikely and we could always update if the scenario does come up. Thank you